codec-http2/src/main/java/io/netty/handler/codec/http2/Http2LocalFlowController.java

```diff
@@ -24,20 +24,19 @@
 public interface Http2LocalFlowController extends Http2FlowController {
 
     /**
-     * Receives an inbound {@code DATA} frame from the remote endpoint and applies flow control
-     * policies to it for both the {@code stream} as well as the connection. If any flow control
-     * policies have been violated, an exception is raised immediately, otherwise the frame is
-     * considered to have "passed" flow control.
+     * Receives an inbound {@code DATA} frame from the remote endpoint and applies flow control policies to it for both
+     * the {@code stream} as well as the connection. If any flow control policies have been violated, an exception is
+     * raised immediately, otherwise the frame is considered to have "passed" flow control.
      * <p/>
-     * If {@code stream} is closed, flow control should only be applied to the connection window.
+     * If {@code stream} is {@code null} or closed, flow control should only be applied to the connection window and the
+     * bytes are immediately consumed.
      *
      * @param ctx the context from the handler where the frame was read.
-     * @param stream the subject stream for the received frame. The connection stream object must
-     *            not be used.
+     * @param stream the subject stream for the received frame. The connection stream object must not be used. If {@code
+     * null} or closed, only the connection window is impacted.
```

Comment:
Use the same text as below.

`If {@code stream} is {@code null} or closed (i.e. {@link Http2Stream#state()} method returns {@link Http2Stream.State#CLOSED}), ...`